apps/server/src/modules/demo/service/demo-school.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@nestjs/common';
+import { ObjectId } from 'bson';
+import { DemoSchool } from '../domain';
+import { DemoSchoolRepo } from '../repo/demo-school.repo';
+
+@Injectable()
+export class DemoSchoolService {
+	constructor(private readonly repo: DemoSchoolRepo) {}
+
+	async createSchool(): Promise<DemoSchool> {
+		const school = new DemoSchool({
```

Comment:
Include a name with a **DEMO_** prefix.
Because in case we want to find the demo schools later we have a easier time :)